zulipterminal/ui.py

```diff
@@ -315,6 +316,25 @@ def mouse_event(
 
         return super().mouse_event(size, event, button, col, row, focus)
 
+    def render(self, size: urwid_Box, focus: bool) -> Any:
+        maxcols, maxrows = size
+
+        if not self.autohide:
+            if maxcols <= 102 and self.mode != "small":
+                self.mode = "small"
+                if self.focus_col != 2:
```

Comment:
Are these checks why you're setting self.focus_col? They have no other functional purpose?

Reply:
They replace every `self.body.focus_position` there are about 17+ of those.